spec/defines/nexus_spec.rb

```diff
@@ -44,7 +44,8 @@
         extract: true,
         extract_path: '/opt',
         creates: '/opt/artifact/WEB-INF',
-        cleanup: true
+        cleanup: true,
+        temp_dir : '/tmp'
```

Comment:
this is not how [symbolised hashes in Ruby](https://ruby-doc.org/core-2.7.1/Hash.html) work
there must not be a space between the word, and the `:`

Reply:
Thx, I have fixed it